api4/team_test.go

```diff
@@ -1691,8 +1509,9 @@ func TestAddTeamMembers(t *testing.T) {
 		otherUser.Id,
 	}
 
-	if err := th.App.RemoveUserFromTeam(th.BasicTeam.Id, th.BasicUser2.Id, ""); err != nil {
-		t.Fatalf(err.Error())
+	err := th.App.RemoveUserFromTeam(th.BasicTeam.Id, th.BasicUser2.Id, "")
+	if err != nil {
```

Comment:
You can replace it with an `require.Nilf(t, err, "success expected, error %v found", err)`